File: CS50/c/Lacture 0 & 1/agree.c
#include <stdio.h>
int main(void)
{
    char answer[10];
    printf("Do you agree? ");
    scanf("%s", answer);
    if(answer[0] == 'y' || answer[0] == 'Y')
    {
        printf("Agreed!\n");
    }
    else if(answer[0] == 'n' || answer[0] == 'N')
    {
        printf("Not agreed!\n");
    }
    else
    {
        printf("Invalid input! Please enter 'yes' or 'no'.\n");
    }
    return 0;
}

File: CS50/Python/agree.py
s = input("Do you agree ? ")

# if s == "Y" or s == "y":
#     print("Agree")
# elif s == "N" or s == "n":
#     print("Not agree")

if s in ["Y", "y", "yes"]:
    print("Agree")
elif s in ["N", "n", "no"]:
    print("Not agree")
else:
    print("Invalid")